ch_tools/common/result.py

```diff
@@ -15,10 +18,10 @@ def __init__(self, code=OK, message="OK", verbose=""):
 class Status:
     """Class for holding Juggler status."""
 
-    def __init__(self):
+    def __init__(self) -> None:
         self.code = 0
-        self.text = []
-        self.verbose = []
+        self.text: List[str] = []
```

Comment:
`List[str]` is deprecated in favor of `list[str]`

https://docs.python.org/3/library/typing.html#typing.List

Reply:
Fixed